OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -26,14 +26,14 @@ public class CarryallInfo : ITraitInfo, Requires<BodyOrientationInfo>, Requires<
 		[Desc("Delay on the ground while attaching an actor to the carryall.")]
 		public readonly int LoadingDelay = 0;
```

Comment:
Can we please rename this to `BeforeLoadDelay` to match `Cargo`?

Reply:
Done.